views/otherProfile.hbs

```diff
@@ -0,0 +1,57 @@
+<script src="/js/gallery.js"></script>
+
+{{> nav}}
```

Comment:
partials should be inside "main.hbs"

Reply:
Nope. landing, login and signup hbs also loads the main layout. We wouldn't want the navbar to be in the public views.

Reply:
oks